### ISSUE TITLE
apply some form of fuzz testing to the package functions

### ISSUE THREAD

**Issue by mcol** [R-Lum/Luminescence#439]
A way to harden the package against misspecified inputs is by calling all functions with a variety of given inputs and record the failures. Rather than concentrating on a large number of random inputs, for our needs at this stage it's sufficient to just test our large number of functions on the same inputs to reveal lacks in our input validation or elsewhere.

I've written this simple package, to implement this: https://mcol.github.io/caught-by-the-fuzz/

This takes advantage of the fact that when we have handled an error, we report the function name in the error message. So by ignoring those errors, we are left with the unhandled errors. These are effectively already unit tests that could be added to our test suite.

This is meant as a tracking bug for this work.

**Comment by RLumSK** [comment]
👏 You didn't tell me that you wrote a package, you just said a few lines ... :D

**Comment by mcol** [comment]
It's literally a few lines (~50 at the moment). The package is just for convenience of installation! :)

**Comment by RLumSK** [comment]
No, but great, this fits actually perfect into what we discussed earlier!
